tests/e2e/local/minikube/setup_host.sh

```diff
@@ -53,6 +53,11 @@ if [[ -z "${ISTIO// }" ]]; then
 fi
 
 #Setup LocalRegistry
+if [ ! -f $ISTIO/istio/tests/util/localregistry/localregistry.yaml ]; then
+    echo File $ISTIO/istio/tests/util/localregistry/localregistry.yaml not found!.
+    echo Please check $ISTIO path is pointing to your Istio codebase.
```

Comment:
Please make sure $ISTIO points to your Istio codebase. See https://github.com/istio/istio/wiki/Dev-Guide.

Reply:
Thanks.. Fixed